src/main/java/duke/UI.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+public class UI {
```

Comment:
This class is underutilised considering so many print statements were used in your other classes. Do let UI class take care of those message handling as it is relevant to it!